scapy/arch/windows/compatibility.py

```diff
@@ -97,7 +97,7 @@ def sndrcv(pks, pkt, timeout = 2, inter = 0, verbose=None, chainCC=0, retry=0, m
                 remaintime = None
                 try:
                     try:
-                        while 1:
+                        while True:
```

Comment:
Doesn't look so, but it unifies how scapy's code is written.
It was by default in python's `lib2to3` and they say it's a recommanded change:
https://docs.python.org/3.1/library/2to3.html#2to3fixer-idioms